ios/FluentUI.Demo/FluentUI.Demo/Demos/PersonaBadgeCollectionDemoController.swift

```diff
@@ -0,0 +1,138 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class PersonaBadgeCollectionDemoController: UITableViewController {
+    override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+        super.init(style: .grouped)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: PersonaBadgeCollectionDemoController.largeBadgeReuseIdentifier)
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: PersonaBadgeCollectionDemoController.smallBadgeReuseIdentifier)
+
+        tableView.separatorStyle = .none
+    }
+
+    // MARK: - Table view data source
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 2
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        var cell: UITableViewCell
+        if showsFullGrid(at: indexPath) {
+            fatalError("showsFullGrid() should always return false right now!")
+        } else {
+            cell = self.tableView(tableView, badgeCellForRowAt: indexPath)
+        }
+
+        return cell
+    }
+
+    override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        switch personaBadgeSize(for: section) {
+        case .small:
+            return "Small"
+        case .large:
+            return "Large"
+        }
+    }
+
+    // MARK: - Helpers
+
+    private func personaBadgeSize(for section: Int) -> MSFPersonaBadgeSize {
+        return (section % 2 == 0) ? .small : .large
+    }
+
+    private func showsFullGrid(at indexPath: IndexPath) -> Bool {
+        // TODO
+        return false
+    }
+
+    private func tableView(_ tableView: UITableView, badgeCellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        var cell: UITableViewCell
+        let size = personaBadgeSize(for: indexPath.section)
+        switch size {
+        case .large:
+            cell = tableView.dequeueReusableCell(withIdentifier: PersonaBadgeCollectionDemoController.largeBadgeReuseIdentifier, for: indexPath)
+        case .small:
+            cell = tableView.dequeueReusableCell(withIdentifier: PersonaBadgeCollectionDemoController.smallBadgeReuseIdentifier, for: indexPath)
+        }
+
+        // Create the PersonaBadge to be displayed
+        let personaBadge = MSFPersonaBadgeView(size: size)
+        let persona = personas[indexPath.item]
+        personaBadge.state.image = persona.avatarImage
```

Comment:
Are you reproducing this on iOS13?

![Screen Shot 2021-06-11 at 10 14 42 AM](https://user-images.githubusercontent.com/68076145/121725227-d8265880-ca9d-11eb-982b-3bc9c5c322c3.png)

Reply:
I do 😿  Very strange.

I'll start looking at it, but if this is something we've seen before I'd be happy for any advice 🙇

Reply:
Yep, it was a failure of rendering mode. The `Image` inside `AvatarView` was defaulting to the `template` rendering mode when I embedded it in a `Button`, but only on iOS 13. Adding an explicit `.renderingMode(.original)` resolves the issue.

Thanks again for the catch!